scripts/linters/pylint_extensions_test.py

```diff
@@ -3257,3 +3257,67 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class DisallowBlankLinesBelowFunctionDefinitionCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(
+            DisallowBlankLinesBelowFunctionDefinitionCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.DisallowBlankLinesBelowFunctionDefinitionChecker)
+        self.checker_test_object.setup_method()
+
+    def test_empty_line_below_function_definition_raises_error(self):
+        node_empty_line_below_function_def = astroid.scoped_nodes.Function(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                def sum(a,b):
+
+                    \"\"\" this  does something \"\"\"
+                    return a+b
+                """)
+        node_empty_line_below_function_def.file = filename
+        node_empty_line_below_function_def.path = filename
+        node_empty_line_below_function_def.fromlineno = 2
+
+        self.checker_test_object.checker.visit_functiondef(
+            node_empty_line_below_function_def)
+
+        message = testutils.Message(
+            msg_id='remove-blank-lines-below-function-definiton.',
+            node=node_empty_line_below_function_def)
+
+        with self.checker_test_object.assertAddsMessages(message):
+            temp_file.close()
+
+    def test_empty_line_below_function_definition_raises_no_error(self):
```

Comment:
Please correct the test name - `test_no_empty_line_below_function_definition_raises_no_error `

Reply:
Done!

Reply:
> Please correct the test name - `test_no_empty_line_below_function_definition_raises_no_error `